.github/scripts/pipeline/attach_cards.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+attach_device()
+{
+	DOMAIN=$1
+	PCI_ID=$2
+	XML_FILE=$3
+            #####################  Creating xml file of the pci device  #######################
+
+	virsh nodedev-dumpxml --device $PCI_ID > temp.xml
+	BUS0=$(xmllint --xpath 'string(//device/capability/bus)' temp.xml)
+	SLOT0=$(xmllint --xpath 'string(//device/capability/slot)' temp.xml)
+	FUNCTION0=$(xmllint --xpath 'string(//device/capability/function)' temp.xml)
+	xmlstarlet ed -L -u '//hostdev/source/address/@bus' -v "$BUS0" $XML_FILE
+	xmlstarlet ed -L -u '//hostdev/source/address/@slot' -v "$SLOT0" $XML_FILE
+	xmlstarlet ed -L -u '//hostdev/source/address/@function' -v "$FUNCTION0" $XML_FILE
+	rm -f temp.xml
+
+	virsh attach-device $DOMAIN $XML_FILE --live
+	sleep 5s
```

Comment:
Once virsh attach-device command is executed, enclave host sends command to guest VM to attach card. 
But guest VM takes the decision when to accept pcie card attachment i.e., it might not happen right away. Hence a sleep of 5s is introduced.

However, in the next commit, sleep command will be replaced by virsh event command which is more reliable.